### DISCUSSION TITLE
Can I add DIA files (.d) to a set that has already been run?

### DISCUSSION THREAD

**Discussion by Cliffome** [vdemichev/DiaNN#1106] [Q&A]
I have processed 100 DIA files using MBR. Can I add a few more DIA files (.d) to the batch that has already been processed without having to reprocess ?

**Comment by vdemichev** [comment]
Hi, 

Can select the 'Use .quant files' checkbox. Then the first pass will not need to be rerun, the second pass will however need to be rerun.

Best,
Vadim

**Comment by Cliffome** [comment]
Thank you Vadim!
